src/ai_migrate/migrate.py

```diff
@@ -265,7 +262,12 @@ def combine_examples_into_conversation(
 
 
 async def call_llm(
-    client: DefaultClient, messages: list, tools: list[Tool], temperature=0.1
+    client: DefaultClient,
+    messages: list,
+    tools: list[Tool],
+    *,
+    context: MigrationContext,
+    temperature=0.1,
```

Comment:
just curious why temperature is now a keyword arg?

Reply:
There isn't really a hard rule here, but once you start having several args I think it's good practice to make most of them keyword-only. Not to mention if you call this particular one without using the keyword, the value is pretty confusing: `call_llm(messages, tools, 0.2)` for example.

Reply:
fair